app/schemas/models/level_session.coffee

```diff
@@ -37,8 +37,6 @@ _.extend LevelSessionSchema.properties,
     type: 'string'
   levelID:
     type: 'string'
-  multiplayer:
```

Comment:
We'll need to unset multiplayer on existing level sessions so they still pass validation after this change.